File: DSA/VaniJain_590012020/Day30Question2.java
import java.util.*;
public class ToggleBitAtPosition
  {
    public static void main(String[] args) 
  {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number (n): ");
        int n = sc.nextInt();
        System.out.print("Enter the bit position to toggle (i): ");
        int i = sc.nextInt();
        int mask = (1 << i);  
        int result = n ^ mask; 
        System.out.println("Number after toggling " + i + "-th bit: " + result);
    }
}

File: DSA/Ayush_Mishra_590014844/Day_30_Question_2.c
#include <stdio.h>

int main() {
    int n, i;
    printf("Enter the number (n): ");
    scanf("%d", &n);
    printf("Enter the bit position to toggle (i): ");
    scanf("%d", &i);
    int result = n ^ (1 << i);
    printf("Result after toggling %d-th bit: %d\n", i, result);
    return 0;
}
